packages/fluentui/docs/src/examples/components/Avatar/Variations/AvatarExampleIcon.shorthand.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```

Comment:
Should be this example in `Slots` section?

Reply:
Probably, but then we should restructure all examples, as we don't even have slots examples here. I will just leave it now after the image example, and we can restructure the examples in a separate PR.